README.md

```diff
@@ -360,6 +360,7 @@ Available letter casing settings are:
 Available whitespace settings are:
 ```ruby
 :trim     # remove whitespace from both ends of the input
+:strip    # idem
```

Comment:
What is `idem`? Could we expand the param behaviour to a simple sentence, please?

Reply:
Sorry, 'idem' "the same as above".
We could use `# remove whitespace from both ends of the input` (as above)

Reply:
Can we say then 'same as :trim' or similar? I prefer plain English to acronyms that not many people may understand including myself 😄

Reply:
Sorry, I'm french and 'idem' is a current word (not a "acronym").
'same as :trim' sounds perfect! :-D